### ISSUE TITLE
fsh and vsh are unrecognised

### ISSUE THREAD

**Issue by dotModded** [AtomLinter/linter-glsl#24]
Unknown shader type for vsh and fsh files

**Comment by andystanton** [comment]
Hi @dotModded 

tl;dr this is fixed in linter-glsl 2.0.1 so if you upgrade `.vsh` and `.fsh` files will lint again.

Removing support for `.vsh` and `.fsh` files was intentional - the [glslangValidator docs](https://www.khronos.org/opengles/sdk/tools/Reference-Compiler/) state that valid shader filename extensions are:

> .vert - a vertex shader
.tesc - a tessellation control shader
.tese - a tessellation evaluation shader
.geom - a geometry shader
.frag - a fragment shader
.comp - a compute shader

So extensions other than these got dropped in linter-glsl 2.0.0.

However, I then realised this was a bad idea and many projects use different shader filename extensions so I added support back for the extensions that were in 1.0.x and a couple of new ones.

The list of supported extensions can be found [here](https://github.com/AtomLinter/linter-glsl#supported-filename-formats)

Cheers,

-Andy

**Comment by dotModded** [comment]
.glsl files are also Unknown

**Comment by dotModded** [comment]
Sorry never mind. just needs a .f. or .v.
